src/utils/BinanceUtils.ts

```diff
@@ -193,7 +194,7 @@ export async function getBinanceWithdrawals(
   }
   const withdrawHistory = Object.values(_withdrawHistory);
   return mapAsync(withdrawHistory, async (withdrawal) => {
-    const provider = withdrawal.network === DepositNetwork.Ethereum ? l1Provider : l2Provider;
+    const provider = withdrawal.network === "ETH" ? l1Provider : l2Provider;
```

Comment:
I'd also like to suggest this, but the naming `DepositNetworks` is weird. Should we call it `BinanceNetworks´ or something?

```suggestion
    const provider = withdrawal.network === DepositNetworks[CHAIN_IDs.MAINNET] ? l1Provider : l2Provider;
```